pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
> Calling it "agile" is just using a buzzword to justify creating technical debt by not considering the general storage picture. 

Don't undermine my response by criticising the term agile. I could easily do the same for use of 'technical debt'. That doesn't further the discussion.

I didn't throw out a buzz word and leave it at that. I clearly outlined exactly what I meant by the term and stated the immediate tangible benefits.

> Adding code into the core to support every different approach does not make sense - these should be done as pluggable storage technologies via a generic framework to enable arbitrary third party implementations, of which this docker container approach is just one such impl.

In time. We have a real need for this feature now. In the future we will adapt as the generic storage abstraction progresses. We aren't there yet though.